representing-data/representing-data-in-lists-and-dictionaries.md

```diff
@@ -114,9 +114,19 @@ Dictionaries are great at...
 | <div style="min-width:130px;">Strength</div>                                  | <div style="min-width:280px">Notes</div>                                                                                                                                                                                               | <div style="min-width:250px;">How it's represented in code</div> |
 | ----------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
 | Data must be organized in key-value pairs | All data in dictionaries _must_ be in key-value pairs; there is no way to add a value without specifying a key                                                                                      |  ``` {"key_name": "value", "another_key": 100} ```                            |
-| Data is unordered                         | Dictionaries may sometimes appear to have an order, but there is no consistent ordering of key-value pairs. This is helpful to express that there isn't necessarily priority or importance to order | ``` {"key_123": 10,    "key_200": 0.34,   "key_2": 1}``` Notice how despite there being a numbering convention in these key-value pairs that they are not in a specific order.  |
+| Data is unordered*                         | Dictionaries may sometimes appear to have an order, but there is no consistent ordering of key-value pairs. This is helpful to express that there isn't necessarily priority or importance to order | ``` {"key_123": 10,    "key_200": 0.34,   "key_2": 1}``` Notice how despite there being a numbering convention in these key-value pairs that they are not in a specific order.  |
 | Accessing values by using keys            | All values can be accessed by a key; this might have more meaning than accessing values by index, especially when keys are named intentionally                                                      | We can access values by using square brackets, ```dict_name["key_name"]```    |
 
+<!-- available callout types: info, success, warning, danger, secondary, star  -->
+### !callout-star
```

Comment:
I didn't know about this callout style!

Since it ends up adding a star icon, could we get by without the asterisk below? I can go either way.

Reply:
That was my original plan but it's a different kind of a star. It's a 5 pointed star. I could change it back to 'info' to make it less redundant

Reply:
LOL. I tried sticking the same star on the initial text and unfortunately it looks ridiculous.

As an alernative `callout-` with an "invalid" style (like none, so `callout-none`) still makes a callout, but leaves of the glyph.